src/backend/joanie/core/admin.py

```diff
@@ -18,13 +18,13 @@
 from joanie.core.enums import PRODUCT_TYPE_CERTIFICATE_ALLOWED
 from joanie.core.forms import ProductCourseRelationAdminForm
 
-ACTION_NAME_GENERATE_CERTIFICATES = "generate_certificates"
+ACTION_NAME_GENERATE_CERTIFICATES = "issue_certificates"
```

Comment:
```suggestion
ACTION_NAME_ISSUE_CERTIFICATES = "issue_certificates"
```
to be consistent, we could change the variable name as well

Reply:
Good catch ! 🙏